sympy/utilities/tests/test_lambdify.py

```diff
@@ -1077,7 +1077,7 @@ def test_scipy_fns():
     if not scipy:
         skip("scipy not installed")
 
-    single_arg_sympy_fns = [Ei, erf, erfc, factorial, gamma, loggamma, digamma]
+    single_arg_sympy_fns = [Ei, erf, erfc, factorial, gamma, loggamma, digamma, polygamma]
```

Comment:
I have no idea how this test works, but does this "guarantee" that polygamma works when both arguments are used? (Just considering the variable name...)

Reply:
No, I didn't read the test code carefully enough, next commit now passes for me locally.